chaincode/compute_plan_worker_state.go

```diff
@@ -0,0 +1,194 @@
+package main
+
+import "fmt"
+
+// TryAddIntermediaryModel will reference the model key if the compute plan key
```

Comment:
What is the meaning of `reference` here?
There is a lot going on: we update the worker state, emit a compute plan event.
This documentation should be more explicit.

Reply:
Good catch! This comment was copy pasted from the previous implementation. I'll update it.